### ISSUE TITLE
Feature request: Support ==highlighed== text

### ISSUE THREAD

**Issue by tpulha** [mokeyish/obsidian-enhancing-export#95]

**Comment by FeralFlora** [comment]
Add this argument: `--from=markdown+mark` to the extra arguments field.

`+mark` enables highlight support.

**Comment by mokeyish** [comment]
> Add this argument: `--from=markdown+mark` to the extra arguments field.
> 
> `+mark` enables highlight support.

Is there any problem with directly modifying the built-in command template? I don't have time to test, if there is no problem, you can change it.

**Comment by machado-t** [comment]
What about exporting highlighted text to docx files?

**Comment by machado-t** [comment]
What about exporting highlighted text to docx files?

**Comment by FeralFlora** [comment]
> > Add this argument: `--from=markdown+mark` to the extra arguments field.
> > 
> > `+mark` enables highlight support.
> 
> Is there any problem with directly modifying the built-in command template? I don't have time to test, if there is no problem, you can change it.

No, that the better way to do it. Just add `+mark` to the main command in the template where you want highlight support.

**Comment by FeralFlora** [comment]
> What about exporting highlighted text to docx files?

Same thing, just add `+mark`.
